templates/role.tmpl

```diff
@@ -0,0 +1,66 @@
+{{define "title"}}Internal: Role ({{.Role.Name}}){{end}}
+{{define "content"}}
+    <div class="column is-10" style="min-height: 88vh">
+        <section class="hero is-info welcome is-small">
+            <div class="hero-body">
+                <div class="container">
+                    <h1 class="title">{{.Role.Name}}</h1>
+                </div>
+            </div>
+        </section>
+        <br>
+        <div class="columns box" style="height: fit-content">
+            <div class="column is-2">
+                <div class="buttons" style="display: block">
+                    <a class="button is-warning is-outlined">
+                        Edit
+                    </a>
+                    <a class="button is-danger is-outlined" onclick="deleteRoleModal()">
+                        Delete
+                    </a>
+                </div>
+            </div>
+            <div class="column">
+                {{parseRoleIntoHTML .Role .UserPermissions}}
+            </div>
+        </div>
+    </div>
+{{end}}
+
+{{define "modals"}}
+    <div id="deleteRoleModal" class="modal">
+        <div class="modal-background"></div>
+        <div class="modal-content">
+            <div class="box">
+                <article class="media">
+                    <div class="media-content">
+                        <div class="content">
+                            <p class="title">Are you sure you want to delete this role?</p>
+                            <p><strong>This cannot be undone</strong></p>
```

Comment:
Can it not? Surely you can just make a new role and add all the permissions back.

(Perhaps a wider point: I'm not sure what the value of the `this can/cannot be undone` notes is, given that anything can be undone with enough database fiddling if you try hard enough.)

Reply:
I have added that point as it would have to be set back up manually